lib/delete.js

```diff
@@ -1,17 +1,17 @@
-const AWS = require('aws-sdk');
+const { S3Client, DeleteObjectCommand, S3ServiceException} = require('@aws-sdk/client-s3');
 const awsError = require('./error.js');
 const parallel = require('parallel-stream');
 
 module.exports = function(bucket, options) {
   options = options || {};
 
   const s3config = {
-    maxRetries: 10,
-    httpOptions: { connectTimeout: 3000 }
```

Comment:
connectTimeout and requestTimeout would not be the same thing. Please refer to here for example https://github.com/mapbox/aws-logs/blob/main/cdk/lib/forwarders/clients/s3.ts#L39-L43